src/tslint-cli.ts

```diff
@@ -21,11 +21,11 @@ import * as optimist from "optimist";
 import * as path from "path";
 import * as Linter from "./tslint";
 import {
-    CONFIG_FILENAME,
-    DEFAULT_CONFIG,
-    findConfigurationPath,
-    getRulesDirectories,
-    loadConfigurationFromPath,
+CONFIG_FILENAME,
```

Comment:
Hmm, this is a little strange, wonder what the normal way to do things is in this situation?